inference-test-tool/test_inference_boxes.py

```diff
@@ -29,10 +29,15 @@ def generate_images_with_boxes(images, boxes, output_folder):
             ul = box['top_left']
             br = box['bottom_right']
             points = [tuple(ul), (br[0], ul[1]), tuple(br), (ul[0], br[1]), tuple(ul)]
-            draw.line(points, fill="red", width=5)
-            
+            draw.line(points, fill='red', width=5)
+            label = box['label']
+            font = ImageFont.truetype('OpenSans-Regular.ttf', br[1] // 20)
```

Comment:
In this case br[1] // 20 is the font size right? Now `br[1]` is the `y` coordinate of the bottom right point. That means that boxes that are further down on the image will have a bigger text than boxes at the top. This might work fine for classification but let's make it work for bounding boxes as well.

Maybe `dcm.Rows // 20`?

Reply:
I agree that this is not very robust.

I made a slightly different proposal relying on the bounding box size instead of the image size, the rationale being that if in the future we want to support multiple bounding boxes in the same image, the labels should be positioned accordingly.

Reply:
We actually support multiple bounding boxes in the same image

Reply:
The problem I see with using the text size relative to bbox size is that for small boxes with, say 20px height, the text would be 1px height and unreadable. That is why a fixed font size would be better.
Other implementations I have seen show the text above the bbox, instead of inside. In that case there would be no issue if the text is bigger than the box

Reply:
I agree with you that this PR handles poorly all edge cases, as my intent was to focus on the most common use case (IMHO) of creating a single bbox which has the same size as the image. In that specific case, placing the label above the bbox sadly leads to a text lying outside of the image.

The worst case scenario for this PR would be having a text label being too small to read, somewhat equivalent to running the inference without this PR. What do you think of keeping it this way as I don't see any negative effect?

Reply:
I see your point regarding placing the label on top of the bounding box. You are right there. However, we are using bboxes for classification as a workaround currently. Ideally we should agree on a format for classification which does not need a bbox. @bsblaw What do you think?